7_react/react1/week1/src/todo_list/TodoList.jsx

```diff
@@ -0,0 +1,51 @@
+import { TodoItem } from "../todoItem/TodoItem";
+const AppTodo = () => {
+    
+    const todos = [
```

Comment:
You have not learned about React's lifecycle methods yet, but just for now take this as a fact, please. Leaving static variables inside of React components is considered bad practice, take it as foreshadowing of upcoming lectures :) Nevertheless, the fix is super simple! Just move the `todos` variable outside of the render as this:

```
const todos = [...some data]
const AppTodo = () => {
```